config/reader.go

```diff
@@ -298,43 +426,39 @@ func ReadLocalConfig(
 	filePath string,
 	logger golog.Logger,
 ) (*Config, error) {
-	buf, err := envsubst.ReadFile(filePath)
+	buf, err := os.ReadFile(path.Clean(filePath))
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "could not read the file")
 	}
-
-	return fromReader(ctx, filePath, bytes.NewReader(buf), logger, false)
+	return fromReader(ctx, filePath, buf, logger, false)
 }
 
-// FromReader reads a config from the given reader and specifies
+// FromReader reads a config from the given byte stream and specifies
```

Comment:
see comment above about name change: should probs change this to something else but wasn't sure if that would break a ton downstream and wanted to minimize the diff